services/matrix/matrix.tester.js

```diff
@@ -439,7 +521,36 @@ t.create('specify the homeserver fqdn')
     color: 'brightgreen',
   })
 
-t.create('test on real matrix room for API compliance')
+t.create('specify the homeserver fqdn for summary')
+  .get('/ALIAS:DUMMY.dumb.json?server_fqdn=matrix.DUMMY.dumb&fetchMode=summary')
+  .intercept(nock =>
+    nock('https://matrix.DUMMY.dumb/')
+      .get(
+        '/_matrix/client/unstable/im.nheko.summary/rooms/%23ALIAS%3ADUMMY.dumb/summary',
+      )
+      .reply(
+        200,
+        JSON.stringify({
+          num_joined_members: 4,
+        }),
+      ),
+  )
+  .expectBadge({
+    label: 'chat',
+    message: '4 users',
+    color: 'brightgreen',
+  })
+
+t.create('test on real matrix room for guest API compliance')
+  .get('/ndcube:openastronomy.org.json?server_fqdn=openastronomy.modular.im')
+  .timeout(10000)
+  .expectBadge({
+    label: 'chat',
+    message: Joi.string().regex(/^[0-9]+ users$/),
+    color: 'brightgreen',
+  })
+
+t.create('test on real matrix room for summary API compliance')
   .get('/twim:matrix.org.json')
   .timeout(10000)
```

Comment:
The `.timeout()`s were here to deal with the live test on the guest account method being slow

We should be able to bin this now and use the default timeout (which I think is 4 seconds from memory). Certainly for the matrix.org/summary case, anyway. Maybe we still need it on the guest case?

Reply:
Done 1ee588cbcf3cb9fef10360abea7b67039b57dd96